src/config.js

```diff
@@ -37,7 +44,7 @@ module.exports.configs = {
       sourceType: 'module',
       ecmaFeatures: { jsx: true }
     },
-    env: {
+    environment: {
```

Comment:
This property needs to be called `env` because it gets passed to ESLint: https://eslint.org/docs/user-guide/configuring#specifying-environments

The ESLint rule `unicorn/prevent-abbreviations` causes way more issues than it's worth, if you ask me. It might make sense to disable it

Reply:
> The ESLint rule unicorn/prevent-abbreviations causes way more issues than it's worth, if you ask me. It might make sense to disable it

I can get behind this take. 🙂

Reply:
Good call! I reverted the changes, disabled that rule, and re-ran the lint fix. Much fewer changes now.